packages/calcite-components/src/components/action-menu/action-menu.e2e.ts

```diff
@@ -237,17 +237,17 @@ describe("calcite-action-menu", () => {
       await page.waitForChanges();
 
       expect(await actionMenu.getProperty("open")).toBe(true);
-      expect(await actions[0].getProperty("active")).toBe(true);
-      expect(await actions[1].getProperty("active")).toBe(false);
-      expect(await actions[2].getProperty("active")).toBe(false);
+      expect(actions[0].getAttribute(activeAttr)).toBe("");
```

Comment:
For later, but this might be better off as a screenshot test since this is now an internal concern.

Reply:
Screenshot tests will already capture the styles for this but this is a nice to test the keyboard navigation.